tests/conftest.py

```diff
@@ -68,40 +68,45 @@ def docs_schema_ids():
     return result
 
 
+def _get_tag(schema):
+    if "tag" in schema:
+        return schema["tag"]
+    elif "anyOf" in schema:
+        for elem in schema["anyOf"]:
+            if "tag" in elem:
+                return elem["tag"]
```

Comment:
It does -- this is just to catch the weird case of unit-1.0.0:

https://github.com/asdf-format/asdf-standard/blob/master/schemas/stsci.edu/asdf/unit/unit-1.0.0.yaml#L19

Reply:
Oh that is weird.  Maybe  comment here might be good for your future self.

What does that empty `{}` even mean in the schema?

Reply:
An empty set of validators never throws an error, so this statement is effectively:

```
assert tag == "tag:stsci.edu:asdf/unit/unit-1.0.0" or True
```

I'm not sure what the point of this is, since the schema only gets used if the tag is present in the first place.

Reply:
Hmm, this where an inline comment in the distant past could have cleared things up.  Thanks.

![Unknown](https://user-images.githubusercontent.com/17088869/118000684-0afcf580-b314-11eb-98f0-6d28a6ff743d.jpeg)